vendor/bat-native-ads/src/bat/ads/internal/notifications.cc

```diff
@@ -112,6 +112,10 @@ bool Notifications::Exists(const std::string& id) const {
   return true;
 }
 
+uint64_t Notifications::Count() const {
```

Comment:
Please change `uint64_t` to an `int` as we will never have that many notifications

Reply:
uint64_t mimics map::size_type, which is unsigned by default. uint64_t is used to satisfy both 32 and 64 bit targets.